apps/docs/components/Navigation/NavigationMenu/NavigationMenu.constants.ts

```diff
@@ -1223,6 +1223,7 @@ export const api: NavMenuConstant = {
         { name: 'Creating API routes', url: '/guides/api/creating-routes' },
         { name: 'How API Keys work', url: '/guides/api/api-keys' },
         { name: 'Securing your API', url: '/guides/api/securing-your-api' },
+        { name: 'Automatic retries', url: '/guides/api/automatic-retries-in-supabase-js' },
```

Comment:
I don't think we want this to be listed for now, right @steve-chavez ?

Reply:
Ideally it should be unlisted, yes. Without a retry header it will be impossible to know which clients are using this feature.

Reply:
Fair enough! Removed the page.